operator/src/test/java/oracle/kubernetes/operator/EventTestUtils.java

```diff
@@ -0,0 +1,151 @@
+// Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.kubernetes.operator;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import javax.validation.constraints.NotNull;
+
+import io.kubernetes.client.openapi.models.V1Event;
+import io.kubernetes.client.openapi.models.V1ObjectMeta;
+import io.kubernetes.client.openapi.models.V1ObjectReference;
+import oracle.kubernetes.operator.helpers.KubernetesTestSupport;
+
+import static oracle.kubernetes.operator.EventConstants.WEBLOGIC_OPERATOR_COMPONENT;
+
+public class EventTestUtils {
+  private static List<V1Event> getEventsWithReason(@NotNull List<V1Event> events, String reason) {
+    return events.stream().filter(event -> reasonMatches(event, reason)).collect(Collectors.toList());
+  }
+
+  /**
+   * Whether an there is an event matches the given reason and namespace.
```

Comment:
extra "an" in "Whether an there"? Same in other methods in this new class.

Reply:
fixed.

Reply:
there are a few other methods with the same typo